packages/contentful--react-apps-toolkit/src/useSDK.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { renderHook, act } from '@testing-library/react-hooks';
+import { useSDK } from './index';
+import ShallowRenderer from 'react-test-renderer/shallow';
+import React from 'react';
+
+let realUseContext: any;
+let useContextMock: any;
+// Setup mock
+beforeEach(() => {
+  realUseContext = React.useContext;
+  useContextMock = React.useContext = jest.fn().mockReturnValue({ sdk: 'mocked-sdk' });
```

Comment:
Why not properly mock `useContext`?

Reply:
I didn't want to add jsx stuff here, so i prefered this way as I only wanted to test the hook functionality and not the context

Reply:
I thought about

```
jest.mock('react', () => {
  return {
    ...jest.requireActual('react'),
    useContext: () => ({ sdk: 'mocked-sdk' })
  }
});
```

That way you would not manipulate a dependency for your test. I see the risk that a failing test might also break other tests.

Reply:
Ah yes, that's better I agree 👍